src/components/forms/pool_actions/InvestForm.vue

```diff
@@ -566,19 +600,11 @@ export default defineComponent({
           minBptOut.value
         );
         console.log('Receipt', tx);
-        txListener(tx.hash, {
-          onTxConfirmed: (tx: TransactionData) => {
-            emit('success', tx);
-            data.amounts = [];
-            data.loading = false;
-          },
-          onTxCancel: () => {
-            data.loading = false;
-          },
-          onTxFailed: () => {
-            data.loading = false;
-          }
-        });
+        if (isPolygon.value) {
```

Comment:
```suggestion
        if (!supportsBlocknative.value) {
```
Can we make this more generic? We'll possibly be on other networks which don't have block stove support soon. I'd probably swap the two branches as well if we make this change.

Reply:
Good point! https://github.com/balancer-labs/frontend-v2/pull/478/commits/6911dbe65b6f2244ca3569e84c5b991d66e89664